python/modules/variable.py

```diff
@@ -0,0 +1,54 @@
+#! /usr/bin/python
```

Comment:
Hm, I wanted to avoid adding too many logs by "just one line" due to all those functions are being used repeatedly during parsing task variables. I'll some logs to VariableParted (before and after concatenation), but I think Var, VarRef i VarList don't require additional msges atm.